docs/java/features/odata/use-vdm.md

```diff
@@ -101,6 +101,25 @@ Request parameters:
    - `ignoreAnyVersionIdentifier()` will instead always send a `*` which acts as a wildcard to match all ETags.
 - All operations allow for adding custom headers via `withheader(...)`
 
+Navigable entities:
+- By using the `forEntity` method on service classes, you can recursively traverse navigation properties in entity sets. It's a generic API extension with type-safe methods to allow for requests along chained entities, according to their navigation properties.
+- Consider the following example:
+  ```java
+  // Create a new Trip in Vacation (id 2020) of Person (username "John")
+  // POST /ODataService/People('John')/Vacations(2020)/Trips
+  
+  Trip tripItem;
+  Person personById = Person.builder().username("John").build();
```

Comment:
Did you consider using the BusinessPartner service? I think it would make things a bit easier to understand since there the datamodel is already known to the reader (at least through the examples contained already).

Reply:
Yeah.. would be better.

Reply:
Changed to _BusinessPartner_, although this is not a good example service since all entities are accessible from service root.